docs/eta-pi-p/manual.ipynb

```diff
@@ -383,6 +615,71 @@
     "fig.tight_layout()\n",
     "plt.show(fig)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Phase space"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phsp = helicity_transformer(phsp_momenta)\n",
+    "list(phsp)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unfolded_expression = intensity_expr.doit().expand(func=True)\n",
+    "intensity_func = create_parametrized_function(\n",
```

Comment:
I now noticed that you already define `intensity_func` in a cell above, directly with `sp.lambdify()`, but that you actually don't use it in the rest of the notebook. Better to remove that one then.